app/api/helpers/mail.py

```diff
@@ -374,6 +374,8 @@ def send_email_to_attendees(order, purchaser_id, attachments=None):
             )
         else:
             # The Ticket holder is not the purchaser
+            start_date = "".join(str(order.event.starts_at.date()).split("-"))
+            end_date = "".join(str(order.event.ends_at.date()).split("-"))
```

Comment:
Use proper library methods to parse and convert date to event timezone

Reply:
ok